gent_base.theme

```diff
@@ -806,7 +806,7 @@ function gent_base_preprocess_node(&$variables) {
 function _gent_base_preprocess_content_header_classes(&$variables) {
   $bundles = ['infopage', 'event', 'news'];
   $node = $variables['node'];
-  $classes = ['content-header-container', 'primary'];
+  $classes = ['content-header-container', 'primary', 'overlap'];
```

Comment:
Let op want die "overlap" is enkel in het geval er paragrafen overlappen met de header. Daar is logica voor voorzien. Dit gaat volgens mij dingen breken (te veel padding waar het niet nodig is).

Reply:
Ja, idd. Heb t daar verwijderd en in het template gestoken nu (detail layout) waar enkel maar een header zonder afbeelding steekt